extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -22,7 +24,7 @@ static public function onGetRailModuleList( &$modules ) {
 		$app = F::App();
 		$pos = $app->wg->User->isAnon() ? 1305 : 1285;
 
-		$modules[$pos] = array( 'Recirculation', 'container', ['containerId' => 'recirculation-rail'] );
+		$modules[$pos] = [ 'Recirculation', 'container', [ 'containerId' => 'recirculation-rail' ] ];
```

Comment:
it's the new short array syntax since php 5.4, preferred by our guidelines https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#array-syntax